app/src/test/java/com/teamagam/gimelgimel/ExampleUnitTest.java

```diff
@@ -1,12 +1,17 @@
 package com.teamagam.gimelgimel;
 
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
 
 /**
  * To work on unit tests, switch the Test Artifact in the Build Variants view.
  */
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = BuildConfig.class, sdk = 21)
```

Comment:
It specifies the SDK Robolectric uses to run the tests under. I think it would be best to test them against th minimum SDK support. As a side-note, Robolectric doesnt support sdk 23 yet.